src/operations.jl

```diff
@@ -17,6 +17,14 @@ function -(op :: AbstractLinearOperator{T}) where T
   LinearOperator{T}(op.nrow, op.ncol, op.symmetric, op.hermitian, prod, tprod, ctprod)
 end
 
+# y ⇽ α * op * x + β * y
+function mul!(y :: AbstractVector, op :: AbstractLinearOperator, x :: AbstractVector, α :: Number, β :: Number)
+  β ≠ 0 && (y .*= β)
+  α ≠ 0 && (y .+= α .* (op * x))
```

Comment:
Does this assume that `op` Is able to directly update `y`? That's how it works with matrices I think.

Reply:
No, one vector is allocated (or already preallocated) for `op * x` product and after  `y` is updated.